aspnetcore/fundamentals/includes/apis6.md

```diff
@@ -0,0 +1,36 @@
+ :::moniker range="= aspnetcore-6.0"
+
+Here's sample code for an API based on controllers:
+
+:::code language="csharp" source="~/fundamentals/apis/APIWithControllers/Program.cs":::
+
+:::code language="csharp" source="~/fundamentals/apis/APIWithControllers/Controllers/WeatherForecastController.cs":::
+
+The following code provides the same functionality in a minimal API project. Notice that the minimal API approach involves including the related code in lambda expressions.
+
+:::code language="csharp" source="~/fundamentals/apis/MinimalAPI/Program.cs":::
+
+Both API projects refer to the following class:
+
+:::code language="csharp" source="~/fundamentals/apis/APIWithControllers/WeatherForecast.cs":::
+
+Minimal APIs have many of the same capabilities as controller-based APIs. They support the configuration and customization needed to scale to multiple APIs, handle complex routes, apply authorization rules, and control the content of API responses. There are a few capabilities available with controller-based APIs that are not yet supported or implemented by minimal APIs. These include:
+
+- No built-in support for model binding (<xref:Microsoft.AspNetCore.Mvc.ModelBinding.IModelBinderProvider>, <xref:Microsoft.AspNetCore.Mvc.ModelBinding.IModelBinder>). Support can be added with a custom binding shim.
+<!--
+- No support for binding from forms. This includes binding <xref:Microsoft.AspNetCore.Http.IFormFile>.
+-->
```

Comment:
Is there a reason to leave it in a comment? I would vote to either delete it or leave it in uncommented if it was true in .NET 6.

Reply:
Copy/paste error. It's not in .NET 6